chapters/functions.tex

```diff
@@ -232,6 +232,7 @@ \section{Pure Modelica Functions}\label{pure-modelica-functions}
   If a function is declared as \lstinline!impure! any function extending from it shall be declared as \lstinline!impure!.
 \item
   A deprecated semantics is that external functions (and functions defined in Modelica directly or indirectly calling them) without \lstinline!pure! or \lstinline!impure! keyword are assumed to be impure, but without any restriction on calling them.
+  Similarly they can be part of systems of equations without restrictions.
```

Comment:
If we manage to write the text clearly, there shouldn't be a need for this sentence, as the systems-of-equations rule should be perceived as one of the calling restrictions.